www/index.md

```diff
@@ -63,6 +63,7 @@ experiences without callback hell
 [Companion of **htmx**](https://htmx.org) |
 [**Comparison** with VanillaJS & jQuery](/comparison) |
 [Read the **docs**](/docs) |
+[Cheatsheet](/hyperscript_cheatsheet_v2021-12-20.pdf) 
```

Comment:
Missing a `|` here

Reply:
@dz4k I omitted the | on purpose, cause it looks weird if there's a pip at the end of the row, with the playground link on the new row. I checked it on desktop and mobile and it seemed ok, but I think there will be some small resolutions where the links will probably all stack vertically and then it does look weird without the pipe.

What do you suggest I do here?

Reply:
Short of a css media query I mean